cmd/kpod/main.go

```diff
@@ -57,6 +58,10 @@ func main() {
 			Usage: "path to the 'run directory' where all state information is stored",
 		},
 		cli.StringFlag{
+			Name:  "runtime",
+			Usage: "path to the OCI-compatible binary used to run containers, default is /usr/bin/runc",
```

Comment:
I wouldn't even bother specifying the default here, given that we're passing it in from a config file - no guarantee it's runc, and we'd have to fetch the configuration file (which can be overridden by flag) and parse it before building flags to get the actual default.